src/components/app.jsx

```diff
@@ -1,9 +1,48 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import CreateCards from './create-cards'
+import ReviewCards from './review-cards'
+import ViewCards from './view-cards'
+import Nav from './nav'
 
 class App extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            view: 'view-cards',
+            cssBadge: ''
+        }
+        this.setView = this.setView.bind(this);
+        this.getView = this.getView.bind(this);
+    }
+
+    setView(newView){
+        this.setState({
+            view: newView,
+            cssBadge: newView
```

Comment:
If `view` and `cssBadge` are always set to the same value you don't need both. You can just use the view property for whatever you are doing with the `cssBadge` property.